src/Microsoft.PowerShell.Commands.Utility/commands/utility/sort-object.cs

```diff
@@ -17,6 +18,17 @@ public sealed class SortObjectCommand : OrderObjectBase
     {
         #region Command Line Switches
         /// <summary>
+        /// This param specifies if a stable sort is required.
+        /// </summary>
+        /// <value></value>
+        [Parameter(ParameterSetName="Default")]
+        public SwitchParameter Stable
+        {
+            get { return _stable; }
+            set { _stable = value; }
+        }
+        private bool _stable;
```

Comment:
Please use auto-property.

Reply:
My preference is absolutely for auto-properties; however, the reason I chose to use this format was simply to follow existing style for other SwitchParameter properties. My assumption is that whoever implemented sort had a reason for creating a boolean field to accompany a SwitchParameter property instead of using an auto-property. If we are to change this cmdlet to use auto-properties for parameters of type SwitchParameter, I think it should be done for all the switch parameters in the cmdlet, not just the one that I have added. Thoughts?

Reply:
You know what, nevermind, I don't see any advantage to using a boolean field with a switchparameter parameter, so I made this change for both -Stable and -Unique parameters.